app/src/main/java/com/example/desafios_zup/view/search/SearchFragment.kt

```diff
@@ -0,0 +1,85 @@
+package com.example.desafios_zup.view.search
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.SearchView
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.desafios_zup.R
+import com.example.desafios_zup.viewModel.SearchViewModel
+import kotlinx.android.synthetic.main.fragment_search.*
+
+class SearchFragment : Fragment() {
+    private lateinit var searchViewModel: SearchViewModel
+    val mAdapter: RecyclerViewSearchAdapter = RecyclerViewSearchAdapter()
+    lateinit var recyclerViewMovies: RecyclerView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        searchViewModel = ViewModelProvider(this).get(SearchViewModel::class.java)
+        var rootView = setup(inflater.inflate(R.layout.fragment_search, container, false))
+
+        val movie_search =
+            rootView.findViewById(R.id.search_view_movies) as androidx.appcompat.widget.SearchView
+        movie_search.setOnQueryTextListener(movieSearch())
+
+        return rootView
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        recyclerViewMovies = recycler_view_movie
+        recyclerViewMovies.layoutManager = LinearLayoutManager(context)
+        recyclerViewMovies.setHasFixedSize(true)
+
+        recyclerViewMovies.adapter = mAdapter
+    }
+
+    private fun setup(view: View): View {
```

Comment:
uma sugestão séria especificar no nome o que você está configurando 
```suggestion
    private fun setupSearchView(view: View): View {
```

Reply:
done